semanticdb/metap/src/main/scala/scala/meta/internal/metap/SymbolInformationPrinter.scala

```diff
@@ -9,7 +9,21 @@ import scala.meta.internal.semanticdb.SymbolInformation._
 import scala.meta.internal.semanticdb.SymbolInformation.Kind._
 import scala.meta.internal.semanticdb.SymbolInformation.Property._
 
-trait SymbolInformationPrinter extends BasePrinter {
+trait SymbolInformationPrinter extends BasePrinter with ConstantPrinter { self =>
+
+  protected def printInfosVisited(infos: List[SymbolInformation]): Unit = {
+    val printed = mutable.Set[String]()
+    infos.foreach { info =>
+      if (!printed(info.symbol)) {
+        printed += info.symbol
+        out.print("  ")
+        out.print(info.name)
+        out.print(" => ")
+        out.println(info.symbol)
+      }
+    }
+  }
```

Comment:
I don't think that sharing 10 lines of straightforward code is worth: 1) increased coupling between `SymbolInformationPrinter` and `SyntheticPrinter`, 2) loss of flexibility to modify `Symbols:` and `Synthetics:` printers independently, 3) the hassle to come up with a name. I suggest we inline this method into its callers.